magicctapipe/scripts/lst1_magic/semi_automatic_scripts/config_auto_MCP.yaml

```diff
@@ -1,15 +1,9 @@
-mc_tel_ids:
-    LST-1: 1
-    LST-2: 0
-    LST-3: 0
-    LST-4: 0
-    MAGIC-I: 2
-    MAGIC-II: 3
-
 directories:
     workspace_dir: "/fefs/aswg/workspace/elisa.visentin/auto_MCP_PR/"  # Output directory where all the data products will be saved.
     MC_version: "" #MCP version used to process MC; if not provided, current MCP version will be used
     real_input_version: ""  #MCP version used to process input real data (to be used in case you want to run only the last steps of the analysis, using lower-level data previously processed); if not provided, current MCP version will be used 
+    RF: ''
```

Comment:
in principle you could provide the /fefs/ paths here since those are pretty standard